Dockerfile

```diff
@@ -29,6 +30,9 @@ RUN \
   git \
   zlib-dev \
   brotli-dev \
+  yaml-dev \
+  imagemagick \
+  imagemagick-dev \
```

Comment:
This is not required to build Imagick (if that's the reason, why you install it).

Reply:
Are you talking about `imagemagick`, `imagemagick-dev` or both?

Reply:
Just the dev package.